pathfinder/cli.py

```diff
@@ -7,20 +7,34 @@
 import logging
 import os
 import sys
+from typing import List
 
 import click
 from raiden_libs.blockchain import BlockchainListener
 from raiden_libs.contracts import ContractManager
 from web3 import HTTPProvider, Web3
+from eth_utils import is_checksum_address
 
 import pathfinder
 from pathfinder.no_ssl_patch import no_ssl_verification
 from pathfinder.pathfinding_service import PathfindingService
 from pathfinder.transport import MatrixTransport
+from pathfinder.utils.types import Address
 
 log = logging.getLogger(__name__)
 
 
+def check_supplied_token_network_addresses(token_network_addresses: List[str]) -> List[Address]:
+    result = []
+    for address in token_network_addresses:
+        if not is_checksum_address(address):
+            log.error(f"Token Network address '{address}' is not a checksum address. Ignoring.")
```

Comment:
What's this, what's this!

I must have missed that. I'm going to `f'` all the things now.

Reply:
Go for it 🚀